pre-processing/primod/driver_coupling/metamod.py

```diff
@@ -0,0 +1,117 @@
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Any
+
+from imod.mf6 import GroundwaterFlowModel, Modflow6Simulation
+from imod.msw import GridData
+
+from primod.driver_coupling.driver_coupling_base import DriverCoupling
+from primod.mapping.node_svat_mapping import NodeSvatMapping
+from primod.mapping.rch_svat_mapping import RechargeSvatMapping
+from primod.mapping.wel_svat_mapping import WellSvatMapping
+
+
+@dataclass
```

Comment:
I think we should go for pydantic instead

Reply:
I concur